dask/dataframe/io/parquet/arrow.py

```diff
@@ -1123,11 +1125,13 @@ def _create_dd_meta(cls, dataset_info, use_nullable_dtypes=False):
 
         # Use _arrow_table_to_pandas to generate meta
         arrow_to_pandas = dataset_info["kwargs"].get("arrow_to_pandas", {}).copy()
+        convert_strings = dataset_info["kwargs"].get("convert_strings", False)
```

Comment:
Should this be `"convert_string"`? Same question below.

Reply:
"convert_strings" sounds more English to me, but I don't have a strong opinion. I can change it to match what we have in config.